### PR TITLE
LPD-32065 Remove LPD-19465 FF about Data Set sorting

### PR DESCRIPTION
**Pull request by thektan**
https://liferay.atlassian.net/browse/LPD-32065

- Removes the feature flag for the new FDSSort data structure and DSM sorting tab
- Rebased on top of https://github.com/brianchandotcom/liferay-portal/pull/152764 so the new PW tests would be tested as well

### CODE REVIEW

**Comment by thektan** [comment]
ci:test:sf

**Comment by thektan** [comment]
ci:test:playwright

File: modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/sorting.spec.ts [hunk 1]
```diff
@@ -62,6 +62,160 @@ test.describe('Sorting in Data Set Manager', () => {
...
+
+		await test.step('Click on delete button to check that delete button exists and is clickable', async () => {
+			await page.getByRole('button', {name: 'Delete'}).click();
+		});
+
+		await test.step('Wait for success message to be displayed', async () => {
+			await expect(
+				page.getByText('Success:Your request completed successfully.')
+			).toBeVisible();
+		});
```
Comment (markocikos) [inline modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/sorting.spec.ts:117]: We can use [waitForSuccessAlert](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/utils/waitForSuccessAlert.ts) util here.

**Review by markocikos — APPROVED** [review]
@thektan LGTM! Just a small non-blocking suggestion inline

**Comment by thektan** [comment]
ci:test:sf

**Comment by thektan** [comment]
ci:forward

**Comment by thektan** [comment]
Seems like unrelated errors and since the previous stable/relevant tests passed, I'll just run the playright ones just in case and then forward.

**Comment by thektan** [comment]
ci:test:playwright

**Comment by thektan** [comment]
After investigating the results on testray, none of the failures seem related. Will forward.

**Comment by thektan** [comment]
ci:forward:force

**Comment by thektan** [comment]
ci:forward:force

**Pull request closed without merging**